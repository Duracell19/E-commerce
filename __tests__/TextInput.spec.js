import React from 'react';
import {shallow} from 'enzyme';
import SimpleTextInput from '../components/controls/simpleTextInput';

describe('SimpleTextInput', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow( <SimpleTextInput placeholder='Enter Email' />)
            expect(component).toMatchSnapshot()
        });
    });
});
