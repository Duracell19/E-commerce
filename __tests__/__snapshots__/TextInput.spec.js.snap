// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleTextInput Rendering should match to snapshot 1`] = `
<Component
  allowFontScaling={true}
  onChangeText={[Function]}
  placeholder="Enter Email"
  rejectResponderTermination={true}
  style={
    Object {
      "alignSelf": "stretch",
      "borderBottomColor": undefined,
      "borderBottomWidth": 0.8,
      "fontSize": 20,
      "margin": 5,
      "paddingBottom": 10,
      "paddingTop": 10,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
